static/app/components/group/tagFacets/tagFacetsBars.tsx

```diff
@@ -208,17 +208,25 @@ function BreakdownBars({data, maxItems, project, tag}: Props) {
               to={point.url}
               aria-label={t('Add %s to the search query', point.label)}
               onClick={() => {
-                if (tag && project) {
-                  trackAdvancedAnalyticsEvent('issue_group_details.tags.bar.clicked', {
-                    tag,
-                    value: point.label,
-                    platform: project.platform,
-                    is_mobile: isMobilePlatform(project?.platform),
-                    organization,
-                    type: 'bars',
-                  });
-                }
+                trackAdvancedAnalyticsEvent('issue_group_details.tags.bar.clicked', {
+                  tag,
+                  value: point.label,
+                  platform: project.platform,
+                  is_mobile: isMobilePlatform(project?.platform),
+                  organization,
+                  type: 'bars',
+                });
               }}
+              onMouseOver={() =>
```

Comment:
Do we need to debounce this?

Reply:
yeah we should, thanks for pointing out. i was going to do this but forgot lol